src/main/java/com/google/devtools/build/lib/rules/objc/CompilationSupport.java

```diff
@@ -894,7 +870,7 @@ public InstrumentedFilesInfo getInstrumentedFilesProvider(ImmutableList<Artifact
         NestedSetBuilder.<Artifact>emptySet(Order.STABLE_ORDER),
         // The COVERAGE_GCOV_PATH environment variable is added in TestSupport#getExtraProviders()
         NestedSetBuilder.<Pair<String, String>>emptySet(Order.COMPILE_ORDER),
-        !isTestRule,
+        true,
```

Comment:
Can you precede this argument with a comment like the one below that indicates the formal argument name?

Reply:
Done!